### ISSUE TITLE
missing menu for keyboard nav (.k-menu .k-state-focused)

### ISSUE THREAD

**Issue by AndersMad** [telerik/kendo-theme-default#550]
It is not possible to see keyboard navigation - fix:

```sass
.k-menu {
	.k-state-focused {
		background-color: $someColor;
	}
}
```

**Comment by Dimitar-Goshev** [comment]
https://github.com/telerik/kendo-theme-default/issues/326

**Comment by Dimitar-Goshev** [comment]
Not reproducible with R3 2018 (2018.3.911)
